tests/expectations/core/test_expect_column_values_to_be_string_integers_increasing.py

```diff
@@ -0,0 +1,388 @@
+
+    batch_request = RuntimeBatchRequest(
+        datasource_name="my_datasource",
+        data_connector_name="default_runtime_data_connector_name",
+        data_asset_name="my_data_asset",
+        runtime_parameters={"batch_data": df},
+        batch_identifiers={"default_identifier_name": "my_identifier"},
+    )
+    validator = context.get_validator(
+        batch_request=batch_request,
+        create_expectation_suite_with_name="test",
+    )
+
+    result = validator.expect_column_values_to_be_string_integers_increasing(
+        column="a", strictly=False
+    )
+
+    assert result == ExpectationValidationResult(
+        exception_info={
+            "raised_exception": False,
+            "exception_traceback": None,
+            "exception_message": None,
+        },
+        expectation_config={
+            "expectation_type": "expect_column_values_to_be_string_integers_increasing",
+            "kwargs": {
+                "column": "a",
+                "strictly": False,
+                "batch_id": "57175eeb4a8baa7ae63f44c6540eb559",
+            },
+            "meta": {},
+            "ge_cloud_id": None,
+        },
+        meta={},
+        result={"observed_value": [[True], [6]]},
+        success=True,
+    )
+
+
+def test_pandas_strictly_expect_column_values_to_be_string_integers_increasing_success(
+    data_context_with_datasource_pandas_engine,
+):
+    context: DataContext = data_context_with_datasource_pandas_engine
+
+    df = pd.DataFrame({"a": ["0", "1", "2", "3", "4", "9", "11"]})
+
+    batch_request = RuntimeBatchRequest(
+        datasource_name="my_datasource",
+        data_connector_name="default_runtime_data_connector_name",
+        data_asset_name="my_data_asset",
+        runtime_parameters={"batch_data": df},
+        batch_identifiers={"default_identifier_name": "my_identifier"},
+    )
+    validator = context.get_validator(
+        batch_request=batch_request,
+        create_expectation_suite_with_name="test",
+    )
+
+    result = validator.expect_column_values_to_be_string_integers_increasing(
+        column="a", strictly=True
+    )
+
+    assert result == ExpectationValidationResult(
+        exception_info={
+            "raised_exception": False,
+            "exception_traceback": None,
+            "exception_message": None,
+        },
+        expectation_config={
+            "expectation_type": "expect_column_values_to_be_string_integers_increasing",
+            "kwargs": {
+                "column": "a",
+                "strictly": True,
+                "batch_id": "57175eeb4a8baa7ae63f44c6540eb559",
+            },
+            "meta": {},
+            "ge_cloud_id": None,
+        },
+        meta={},
+        result={"observed_value": [[True], [6]]},
+        success=True,
+    )
+
+
+def test_pandas_expect_column_values_to_be_string_integers_increasing_type_failure(
+    data_context_with_datasource_pandas_engine,
+):
+    with pytest.raises(MetricResolutionError):
```

Comment:
I think you should move this `with` down to the `result` at line 134 to be more precise about where we expect the error to happen.

Reply:
Resolved by moving tests to contrib.